application/api/connectiondetail.py

```diff
@@ -41,23 +26,95 @@ def post(self, session):
         """
         try:
             parser = reqparse.RequestParser()
-            parser.add_argument('connection_type',
-                                help='This field cannot be blank',
+            parser.add_argument('connection_reference',
+                                help=APIMessages.PARSER_MESSAGE,
                                 required=True)
             parser.add_argument('case_id_list',
-                                help='This field cannot be blank',
+                                help=APIMessages.PARSER_MESSAGE,
                                 required=True,
                                 type=args_as_list, default=[])
-            parser.add_argument('db_id',
-                                help='This field cannot be blank',
+            parser.add_argument('db_connection_id',
+                                help=APIMessages.PARSER_MESSAGE,
                                 required=True)
             data = parser.parse_args()
             user = session.user_id
             select_connection(data, user)
+
             return api_response("success", True, 200,
                                 APIMessages.RETURN_SUCCESS)
 
         except Exception as e:
             return api_response(False, APIMessages.INTERNAL_ERROR,
                                 STATUS_SERVER_ERROR,
                                 {'error_log': str(e)})
+
+
+class DbConnection(Resource):
+    """
+    Class will be a get call to give all the db_connection_ids
+     associated with the project_id
+    """
+
+    def get(self):
+        """
+        Method will give all the db_connection_ids associated with the
+        project_id which we will pass in the argument
+
+        Returns: give all the db_connection_ids associated with the
+        project_id which we will pass in the argument
+        """
+        try:
+
+            db_connection_detail = reqparse.RequestParser()
+            db_connection_detail.add_argument('project_id', required=False,
+                                              type=int,
+                                              location='args')
+
+            project_id = db_connection_detail.parse_args()
+            project_obj = Project.query.filter_by(
```

Comment:
When you are querying any DB record, make sure you also check is_deleted is False. For now we dont have delete option, but in future, we may need to add this condition. Its better we take care of it now itself.

Reply:
For now, we are not validating this check. We will add it later.